app/hooks/use-query-params.tsx

```
@@ -0,0 +1,95 @@
+      params.delete(key);
+      const query = params.toString();
+      router.push(query ? `${pathname}?${query}` : pathname);
+    },
+    [createParams, pathname, router]
+  );
+
+  /** 다중 쿼리 파라미터 삭제 후 URL 이동 */
+  const deleteParams = useCallback(
+    (keys: string[]) => {
+      const params = createParams();
+      keys.forEach((key) => params.delete(key));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)